src/generate_tangent.jl

```diff
@@ -0,0 +1,13 @@
+rand_tangent(rng::AbstractRNG, x::Integer) = Zero()
+
```

Comment:
How about for things that are not pertubable

```suggestion

rand_tangent(rng::AbstractRNG, x::Union{Symbol, AbstractChar, AbstractString} = DoesNotExist()

```

Reply:
Good shout. Will push a change.

Reply:
Perhaps `Bool` too?